### DISCUSSION TITLE
CKAN as CDP?!

### DISCUSSION THREAD

**Discussion by nesmachny** [ckan/ckan#6757] [Q&A]
Our project looking for right solution for collecting data from free Wi-Fi users and building User Profile for marketing communication purpose. Is CKAN good for it? 
We need a single user profile, which will be constantly updated.

**Comment by frafra** [comment]
IANAL, but collecting personal information without consent could be illegal depending on where you live.
I guess that you can use whatever software you like the most as CDP, even if it was not meant to be used like that initially. Test CKAN and evaluate yourself.

**Reply by nesmachny** [reply]
Hi! My question is not in legal part, but in technical.

**Reply by frafra** [reply]
I replied to the technical part, but I cannot avoid warning you (and whose reading this. as this is a public page) about the potential non-technical issues.
There is a public demo instance of CKAN if you want to try it without installing it.
